client/client.go

```diff
@@ -55,12 +55,12 @@ func New() *Client {
 // raw performs a request and returns the resulting http.Response and
 // error you usually only need to call this directly if you expect the
 // response to not be JSON, otherwise you'd call Do(...) instead.
-func (client *Client) raw(method, path string, body io.Reader) (*http.Response, error) {
+func (client *Client) raw(method, opaque string, body io.Reader) (*http.Response, error) {
```

Comment:
wouldn't something like encodedPathAndQuery be a more meaningful name here? opaque is opaque :)